docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -290,6 +265,15 @@ Release
 
    Wait for the build to complete successfully.
 
+#. Set up ``AWS Access Key ID`` and ``AWS Secret Access Key`` Amazon S3 credentials:
```

Comment:
Note that there is WIP for removing Vagrant as a release dependency which will mean that this is a one time per engineer task.